client.Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG PYTHON_VERSION
+FROM python:${PYTHON_VERSION} AS builder
+ARG PKGVER
+RUN pip wheel --wheel-dir=/wheels --no-cache-dir backend.ai-client==${PKGVER}
+
+FROM python:${PYTHON_VERSION}
+COPY --from=builder /wheels /wheels
+RUN mkdir -p /root/.ssh
```

Comment:
Can you describe about what this line is for?

Reply:
This is to solve the problem that the ssh key is not created because the folder does not exist when using the ssh function of backend.ai.